documentation/src/main/asciidoc/topics/ref_server_datasources.adoc

```diff
@@ -1,8 +1,9 @@
 [id='datasource_config']
-= Data Sources Configuration
+= Datasource Configuration for JDBC Cache Stores
+{brandname} server configuration for datasources is composed of two sections:
 
-{brandname} configuration for data sources is composed of two sections: a *connection factory* which defines how to
-connect to the database and a *connection pool* which defines how to pool and reuse connections:
+* A `connection factory` that defines how to connect to the database.
+* A `connection pool` that defines how to pool and reuse connections.
```

Comment:
This seems like a good place to use an unordered list, which helps break text up and makes it easier for quick reading.